README.md

```diff
@@ -12,23 +12,25 @@ Vircadia is a 3D social software project seeking to incrementally bring about a
 
 ### How to build the Interface
 
-[For Windows](https://github.com/kasenvr/project-athena/blob/master/BUILD_WIN.md)
+[For Windows](https://docs.vircadia.dev/developer/build/BUILD_WIN.html)
 
-[For Linux](https://github.com/kasenvr/project-athena/blob/master/BUILD_LINUX.md)
+[For Mac](https://docs.vircadia.dev/developer/build/BUILD_OSX.html)
 
-[For Linux - Athena Builder](https://github.com/kasenvr/vircadia-builder)
+[For Linux](https://docs.vircadia.dev/developer/build/BUILD_LINUX.html)
```

Comment:
Are the build guides in the repository still considered the "master" copies? (If not then this should be discussed.)

Are the build guides in the repository (automatically) copied over for publication in the user docs (perhaps at time of a Vircadia release)? (It seems not: e.g., the Windows build guide in the user docs is currently out of date.)

I think the README in the repository should continue to point to the build guides in the repository because they will (should) reflect how to build the current code in `master`. And are more likely to be updated in the repository than a separate location.

Reply:
1. Yes, the repository is considered master for the docs as was suggested before. :)

2. They are automatically copied over from the repo when you generate the docs using `make html`

3. That makes sense, I was setting it up so that when the docs expand out to include more and more information, it can easily be accessed. But that change can be made once there is more information to be had.

The docs were out of date due to the make batch file being outdated in using kasen/core instead of master, this is corrected in https://github.com/kasenvr/vircadia-docs-sphinx/pull/55

And the docs have been updated to correctly reflect this: https://docs.vircadia.dev/developer/build/BUILD_WIN.html